api-reference/50 Common/utils/excelExporter/exportDataGrid(options).md

```diff
@@ -4,13 +4,171 @@ module: excel_exporter
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Exports grid data to Excel.
 
 ##### return: Promise<CellRange>
-<!-- Description goes here -->
+A <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise" target="_blank">Promise</a> that is resolved with an object that contains the coordinates of the first and last cells. 
 
 ##### param(options): ExportDataGridProps
-<!-- Description goes here -->
+Export settings.
+
+---     
+To export **DataGrid** using <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> API, reference or import this library as well as <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> library.
```

Comment:
```suggestion
To export the **DataGrid** using <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> API, reference or import this library as well as <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> library.
```
this is an incomplete sentence; how does reference or import this library fit with the beginning of the sentence? are you missing a word? please clarify what you really want to say.

Reply:
Please see these two links for an example of this phrase: [DevExpress+Quill](https://js.devexpress.com/Documentation/Guide/Widgets/HtmlEditor/Overview/) and a [ticket from the support center](https://supportcenter.devexpress.com/Ticket/Details/T208232/dashboarddesigner-how-to-load-a-custom-shp-file-to-the-map-item-and-show-a-tool-tip-for-it). 
And let me please explain, what I mean. To reference is to insert a link(that is also called a hyperreference, you know) that leadse to a site. For instance, if we want to reference ExcelJS library, we would use a link like this: `<script src="https://github.com/exceljs/exceljs">`.
To import is a similar thing, but used in different approaches. To be short, I would say that we reference libraries in jQuery and import the same libraries in Angular, Vue and React. When we import a library, we need to write something like `import ExcelJS from 'exceljs';`

In both ways, the result will be the same. And we go back to beginning: if we want to export the **DataGrid** using ExcelJs library, we need to include this third-party library into our project. Is this explanation suffficient?

Reply:
Are you trying to say:
To export the **DataGrid** using <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> API, reference or import this library along with the <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> library.

Reply:
Yes, exactly. Thank you for your help!

Reply:
To use ExcelJS API to export the DataGrid, reference or import this library and the FileSaver library.